site/src/components/animationControl.js

```diff
@@ -0,0 +1,22 @@
+import html from '../../../index'; // import innerself
+import { connect } from '../reducers/store';
+
+function AnimationControl({ bgAnimation }){
+	let bgCanvas = window._bgCanvas;
+
+	console.log('bgCanvas', bgCanvas);
```

Comment:
Lets get rid of the `console.log`s

Reply:
I'm more than willing to remove this one, but the `hey!` is there to stay.